src/components/feedbackpage/feedbackpage.scss

```diff
@@ -0,0 +1,38 @@
+#line-1, #line-4{
```

Comment:
User more descriptive names. Since we are using scss the names are global, so the `line-1` used here will conflict with another `line-1` used somewhere else. Consider changing it to something like `feedback-line-1`. You can also refer to SASS nesting with ampersand (https://css-tricks.com/the-sass-ampersand/)